clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/AddExecutor.scala

```diff
@@ -43,98 +39,29 @@ class AddExecutor[CI <: ClioIndex](addCommand: AddCommand[CI])
 
     decodedOrError.fold(
       Future.failed, { decoded =>
```

Comment:
I think we can drop the `{ decoded => ... }` and do `addFiles(webClient, _, addCommand.force)`